HelpSource/Classes/OSCdef.schelp

```diff
@@ -22,7 +22,13 @@ argument:: key
 The key at which to store this OSCdef in the global collection. Generally this will be a link::Classes/Symbol::.
 
 argument:: func
-A link::Classes/Function:: or similar object which will respond to the incoming message. When evaluated it will be passed the arguments msg, time, addr, and recvPort, corresponding to the message as an link::Classes/Array:: in the form code::[OSCAddress, other args]::, the time that the message was sent (plus the latency if the message was in a bundle), a link::Classes/NetAddr:: corresponding to the IP address of the sender, and an link::Classes/Integer:: corresponding to the port on which the message was received.
+A link::Classes/Function:: or similar object which will respond to the incoming message. When evaluated it will be passed the following arguments:
+table::
+## msg || message as an link::Classes/Array:: in the form code::[OSC address, args1, arg2, ...]::
+## time || the time received (for messages) / the time sent plus the latency (if the message was in a bundle)
```

Comment:
"the time sent plus the latency" is inaccurate, I think. It's the timestamp in the OSC bundle. SC happens to calculate the timestamp as `logical time + latency` but we can't assume all OSC software does it the same way.

I'd suggest "/ the OSC bundle's timestamp."

Also a bundle may have a non-timestamp:

```
(
OSCFunc({ |msg, time| time.postln }, '/no_ts').oneShot;
NetAddr("127.0.0.1", NetAddr.langPort).sendBundle(nil, ["/no_ts"]);
)

-> 565991666.20289
```

... no matter what time it is. And, unfortunately, this varies from sclang session to session (!) though it's always the same within one session.

Reply:
ah yes. What is time, after all? Just a condition of the possibility of appearances …